src/main/java/frc/robot/Constants.java

```diff
@@ -259,6 +302,8 @@ public static class FieldConstants {
     public static final Translation2d RED_SPEAKER = new Translation2d(16.579342, 5.547868);
     public static final Translation2d BLUE_SPEAKER = new Translation2d(-0.0381, 5.547868);
     public static final double CENTERLINE_X_APPROX = 8;
+    public static final double SPEAKER_TAG_HEIGHT_METERS = 1.45;
+    public static final Rotation2d AMP_DIRECTION = Rotation2d.fromDegrees(90);
```

Comment:
We have RED_AMP_ROTATION and BLUE_AMP_ROTATION.
Both of them are used in SwerveCommandFactory for a command autoTargetAmp.

The AMP direction is different for Blue and red, isn't it? ( THis applies to AMP_DIRECTION and RED/BLUE_AMP_DIRECTION )

There is also a BLUE/RED_SOURCE_ROTATION with associated commands.

This needs to be cleaned up.  Decide on one implementation, and remove the other.  Test the amp to make sure that it is turning correctly

Reply:
The amp is towards the same direction on field for both alliance side (Opposite judges table). Because the robots internal rotation is relative to the x axis and does not change base on alliance color, neither does the target angle.